.github/workflows/examples-linux-standalone.yaml

```diff
@@ -127,7 +127,17 @@ jobs:
                     linux debug ota-requestor-app \
                     out/ota_requestor_debug/chip-ota-requestor-app \
                     /tmp/bloat_reports/
-
+            - name: Build and run OTA example
+              timeout-minutes: 20
+              run: |
+                  scripts/examples/ota_test.sh
```

Comment:
we have separate "run tests" workflows. This workflow seems to be mainly intended for compilation.
Could we run the tests in the test workflow intead?

Reply:
We could not find any documentation or examples that would show to run this scenario differently.

Reply:
We are indeed doing poorly in documentation terms. However this is the feedback I give in PR review: we have separate flows dedicated for 'check that it builds' and 'check that unit tests pass'.

Please place this in the 'test' flow instead of the build one.